### DISCUSSION TITLE
如何定时拨号 周二，周四，周日， 3点半  求大神看看有啥问题没呢

### DISCUSSION THREAD

**Discussion by 160jelili** [istoreos/istoreos#1080] [Q&A]
<img width="575" alt="截屏2023-10-26 21 28 21" src="https://github.com/istoreos/istoreos/assets/149078408/bd1c4388-454f-4b86-a64a-ff2f863e2106">

**Comment by jjm2473** [comment]
istore里面有个定时任务的插件，里面有重启网络的功能
